lib/ansible/modules/cloud/azure/azure_rm_sqldatabase.py

```diff
@@ -0,0 +1,576 @@
+                    self.parameters["restore_point_in_time"] = kwargs[key]
+                elif key == "recovery_services_recovery_point_resource_id":
+                    self.parameters["recovery_services_recovery_point_resource_id"] = kwargs[key]
+                elif key == "edition":
+                    ev = kwargs[key]
+                    if ev == 'web':
+                        ev = 'Web'
+                    elif ev == 'business':
+                        ev = 'Business'
+                    elif ev == 'basic':
+                        ev = 'Basic'
+                    elif ev == 'standard':
+                        ev = 'Standard'
+                    elif ev == 'premium':
+                        ev = 'Premium'
+                    elif ev == 'free':
+                        ev = 'Free'
+                    elif ev == 'stretch':
+                        ev = 'Stretch'
+                    elif ev == 'data_warehouse':
+                        ev = 'DataWarehouse'
+                    elif ev == 'system':
+                        ev = 'System'
+                    elif ev == 'system2':
+                        ev = 'System2'
+                    self.parameters["edition"] = ev
+                elif key == "max_size_bytes":
+                    self.parameters["max_size_bytes"] = kwargs[key]
+                elif key == "requested_service_objective_id":
+                    self.parameters["requested_service_objective_id"] = kwargs[key]
+                elif key == "requested_service_objective_name":
+                    ev = kwargs[key]
+                    if ev == 'basic':
+                        ev = 'Basic'
+                    elif ev == 's0':
+                        ev = 'S0'
+                    elif ev == 's1':
+                        ev = 'S1'
+                    elif ev == 's2':
+                        ev = 'S2'
+                    elif ev == 's3':
+                        ev = 'S3'
+                    elif ev == 'p1':
+                        ev = 'P1'
+                    elif ev == 'p2':
+                        ev = 'P2'
+                    elif ev == 'p3':
+                        ev = 'P3'
+                    elif ev == 'p4':
+                        ev = 'P4'
+                    elif ev == 'p6':
+                        ev = 'P6'
+                    elif ev == 'p11':
+                        ev = 'P11'
+                    elif ev == 'p15':
+                        ev = 'P15'
+                    elif ev == 'system':
+                        ev = 'System'
+                    elif ev == 'system2':
+                        ev = 'System2'
+                    elif ev == 'elastic_pool':
+                        ev = 'ElasticPool'
+                    self.parameters["requested_service_objective_name"] = ev
+                elif key == "elastic_pool_name":
+                    self.parameters["elastic_pool_name"] = kwargs[key]
+                elif key == "read_scale":
+                    ev = kwargs[key]
```

Comment:
If `read_scale` was a bool you could do `ev = "Enabled" if kwargs[key] else "Disabled"